spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -36,16 +36,16 @@
 			+ File.separator + ".m2" + File.separator + "repository");
 
 	/**
-	 * Location of a remote maven repository from which modules will be downloaded, if not available locally.
+	 * Location of comma separated remote maven repositories from which modules will be downloaded, if not available locally.
 	 */
-	private String remoteRepository = "https://repo.spring.io/libs-snapshot";
+	private String remoteRepositories = "https://repo.spring.io/libs-snapshot";
```

Comment:
We should be able to use an array (or List) type for the remoteRepositories field rather than having to handle the comma-delimited String conversion ourselves.